include/deal.II/lac/trilinos_sparse_matrix.h

```diff
@@ -1579,7 +1581,13 @@ namespace TrilinosWrappers
      * distributed. Otherwise, an exception will be thrown.
      */
     template <typename VectorType>
-    void
+    typename std::enable_if<std::is_same<typename VectorType::value_type,
+                                         TrilinosScalar>::value>::type
+    vmult(VectorType &dst, const VectorType &src) const;
+
+    template <typename VectorType>
+    typename std::enable_if<!std::is_same<typename VectorType::value_type,
+                                          TrilinosScalar>::value>::type
```

Comment:
What do you want me to say here more than describing what the `enable_if` is for?

Reply:
Just explain what this does is good enough

Reply:
OK, I am doing that already. :smile:

Reply:
Yeah but you should explain that the function does not return anything because it looks like it returns a very complicated object which is not the case.

Reply:
Done.